libs/build-all.sh

```diff
@@ -104,6 +104,21 @@ diff -r 65efa74ef84a coreconf/config.gypi
          \'include_dirs\': [
 ' | patch "${NSS_SRC_PATH}/nss/coreconf/config.gypi"
 
+# Early return hack to prevent NSPR Android setup
+# which does not work with ndk unified headers and clang.
+echo $'\
+@@ -2662,6 +2662,9 @@
+
+ case "$target" in
+ *-android*|*-linuxandroid*)
++    $as_echo "#define ANDROID 1" >>confdefs.h
++    ;;
++    bobotron)
```

Comment:
bobotron returns

(Did you mean to delete this? Is it somehow being used?)

Reply:
It's actually used so we don't execute the rest of the `case android` code, there's no way to early return from a `case` statement in autoconf scripts :/

Reply:
I could rename `bobotron` to `unreacheable` to make it clearer